scripts/tests

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+(cd packages/main && eslint src/**/*.ts)
+(cd packages/client && eslint src/**/*.ts)
```

Comment:
It turns out that the eslint command I gave you misses problems in `.tsx` files. You can use these commands instead:

```sh
(cd packages/main && eslint src/**/*.ts src/**/*.tsx)
(cd packages/client && eslint src/**/*.ts src/**/*.tsx)
```